### PR TITLE
Fix #251: Consolidate liquibase scripts

### LINKED ISSUE #251: Consolidate liquibase scripts
Consolidate liquibase scripts according to the Wultra conventions.

### PR DESCRIPTION
**Pull request by jandusil**
- Move db creation to pure liquibase

### CODE REVIEW

**Comment by jandusil** [comment]
@zcgandcomp Take a look please if you can, we have discussed this with @banterCZ but we were not sure if this is the desired outcome

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
1) remove LB from the application.properties.
2) use standard paths schema (/docs/db/powerauth-test-server) 
3) modify the Test server Dockerfile to accommodate changes
4) remove the insertion of template data

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,7 @@
 .idea/
 .mvn
 dist/
+
+# Docker image support files
+powerauth-test-server/env.list
+powerauth-test-server/deploy/liquibase
```
Comment (zcgandcomp) [inline .gitignore:25]: unnecessary

File: docs-private/Developer-How-To-Start.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+   Before executing any command, make sure you are in the correct directory.
+
+    ```
+    cd path/to/powerauth-test-server
+    ```
+
+2. **Prepare the environment file**
+
+   Copy the provided `env.list.tmp` located in `power-auth-server/docker` to `power-auth-server/env.list`.
```
Comment (zcgandcomp) [inline docs-private/Developer-How-To-Start.md:26]: non-existing directories. Guide is a good idea in general, but omit it for now.

File: powerauth-test-server/copy_liquibase.sh
Comment (zcgandcomp) [inline powerauth-test-server/copy_liquibase.sh:1]: just do not use the script at all. It's redundant. Use correct path in Dockerfile.

File: powerauth-test-server/deploy/docker-entrypoint.sh [hunk 3]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env sh
 
-liquibase --headless=true --log-level=INFO --changeLogFile=$LB_HOME/data/powerauth-test-server/db.changelog-module.xml --username=$POWERAUTH_SERVER_DATASOURCE_USERNAME --password=$POWERAUTH_SERVER_DATASOURCE_PASSWORD --url=$POWERAUTH_SERVER_DATASOURCE_URL update
-
-nginx
+liquibase --headless=true --log-level=INFO --changeLogFile=$LB_HOME/data/powerauth-test-server/db.changelog-module.xml --username=$POWERAUTH_SERVER_DATASOURCE_USERNAME --password=$POWERAUTH_SERVER_DATASOURCE_PASSWORD --url=$POWERAUTH_TEST_SERVER_DATASOURCE_URL update
```
Comment (zcgandcomp) [inline powerauth-test-server/deploy/docker-entrypoint.sh:3]: non existing environment variable `POWERAUTH_SERVER_DATASOURCE_PASSWORD`

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: powerauth-test-server/copy_liquibase.sh
Comment (jandusil) [reply powerauth-test-server/copy_liquibase.sh:1]: After further conversation, agreed to be useful

**Comment by jandusil** [comment]
Tested and running
image
image

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**